package.json

```diff
@@ -14,6 +14,7 @@
   },
   "build": {
     "appId": "org.rapidsilicon.rapid_power_estimator",
+    "productName": "Raptor Power Estimator",
```

Comment:
It's "Rapid Power Estimator"

Reply:
https://github.com/os-fpga/rapid_power_estimator/pull/84/commits/20bbbbec877b0ba6d3b6d7a21f094afecf2a9fd0